style.css

```
@@ -39,3 +81,50 @@ p {
   box-shadow: 0 0 25px rgba(0, 0, 0, 0.25);
   background: white;
 }
+
+.inputNewTasks{
+  height: 50px;
+  padding: 8px;
+  flex-direction: row;
+  align-items: stretch;
+  display:flex;
+  font-family: 'NEXON Lv1 Gothic OTF';
+  margin-top: auto;
+  box-shadow: 0 0 25px rgba(0, 0, 0, 0.25);
+
+}
+
+
+
+#todoInput{
+  flex: 1; /*specify it for a child if you wnat it to fill the remaining horizontal spaces*/
+  border-radius: 10px;
+  padding: 0.2px;
+  margin-right: 5px;
+  text-align: center;
+  font-size: 1em;
+
+}
+
```

```suggestion
#todoInput::placeholder{
  color: rgba(0, 0, 0, 0.2);
}
```
또한 , [::placeholder 선택자](https://developer.mozilla.org/ko/docs/Web/CSS/::placeholder)로 placeholder도 스타일링 해주실 수 있으세용